hack/patch-csv.sh

```diff
@@ -88,14 +91,14 @@ versions="$( ${YQ} '.versions[].name' "${VERSIONS_YAML_FILE}" )"
 
 for version in ${versions}; do
   version_underscore=${version//./_}
-  for component_name in "${!COMPONENTS[@]}"; do    
+  for component_name in "${!COMPONENTS[@]}"; do
     name="${version_underscore}.${COMPONENTS[$component_name]}"
```

Comment:
Would it be easier to read/understand if you change this to `name="${version_underscore}.${$component_name}"` and revert the other changes?

Reply:
makes sense. made the change